packages/endpoints-organizations-registrar/src/entities/signatories/orchestrators/send-reminders.js

```diff
@@ -51,6 +51,34 @@ const sendReminder = async (
       return;
     }
 
+    const isMaxReminderCount = flow(size, (n) =>
+      gte(n, config.signatoryMaxReminderCount)
+    )(signatoryStatusDoc.events);
+    if (isMaxReminderCount) {
+      await repos.signatoryStatus.addState(
```

Comment:
add both states at once. its irrelevant f the email is sent successfully

Reply:
done, [3f50c05](https://github.com/velocitycareerlabs/velocitycore/pull/179/commits/3f50c053bb2d7ef07266bac1d1932dc469358c07)